deploy/packages/bbp-packages.yaml

```diff
@@ -40,6 +39,7 @@ packages:
       - python
     specs:
       - py-simwriter@0.4.3 ^py-rpy2@2.8.6 ^openjdk
+      - py-entity-management@1.0.9
```

Comment:
```suggestion
```

Shouldn't be present twice, really.

Reply:
I wanted to have a module with python2 in the name. Now it makes only `py-entity-management/1.0.5/python3`. Looks a bit misleading that it can't be used with python2.

Reply:
Or should I just add `- py-entity-management@1.0.5 ^python@:2` next to the other one?

Reply:
Ah, sorry, forgot that the first one is only py3!